applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -42,6 +43,7 @@ <h3 class="signup-confirmation--success">Check your Inbox!</h3>
                 </div>
 
                 <form action="@LinkTo("/email")" method="post" target="newsletterSignup" class="newsletter-card__signup">
+                    <div class="grecaptcha_container"></div>
```

Comment:
I'm not too familiar with twirl templates, is the reason we're not using the recaptcha fragment here because the terms and conditions are not adjacent to the recaptcha form on this page? Wondering, for future modifications, whether it's intuitive for people to know to change both the fragment, but also this specific template too? Maybe there isn't an easy solution to using the fragment everywhere!

Reply:
yes, exactly - the fragment contains both the container and the terms. I think it would be good to split it, so we have a fragment for the container, and a fragment for the terms.